BEIMA.Backend/UserFunctions/UpdateUser.cs

```diff
@@ -67,12 +68,20 @@ public static class UpdateUser
                 }
                 else
                 {
-                    // TODO: Add validation for password length and special characters
+                    // Verify password meets all requirements
+                    if (string.IsNullOrEmpty(updatedUserRecord.Password) ||
```

Comment:
For UpdateUser, it is okay for password to be null or empty. They are not required to enter password if all they want to do is change other properties like FirstName/LastName, etc

Reply:
Oh wait, I see that this is handled in the if/else block surrounding it. GitHub was trimming off that part and I couldn't see it. Regardless, the "string.IsNullOrEmpty" here is redundant since the if block already checks for that.

Reply:
This still allows for empty/null passwords on update. We can only get into this else statement if the password is not null or empty.

Reply:
Fixed ```313159c```